src/libopensc/pkcs15-sec.c

```diff
@@ -420,12 +420,16 @@ int sc_pkcs15_compute_signature(struct sc_pkcs15_card *p15card,
 
 	/* add the padding bytes (if necessary) */
 	if (pad_flags != 0) {
-		size_t tmplen = sizeof(buf);
+		if (flags & SC_ALGORITHM_RSA_PAD_PSS) {
+			// TODO PSS padding
```

Comment:
This will be needed to implement the PSS off the card. Do you plan to have a look into that, or should I have a look into the possibilities here?

Reply:
We could maybe use this: https://github.com/NWilson/OpenSC/commit/42f3199e66796db6f003668e77315203d07f1038